editions/tw5.com/tiddlers/features/Performance Instrumentation.tid

```diff
@@ -0,0 +1,31 @@
+created: 20190316163428191
+modified: 20190316163657710
+tags: Features
+title: Performance Instrumentation
```

Comment:
I think this is here because it's in tiddlywiki-com but hasn't been merged into master? I've just pulled the latest changes across so hopefully this will disappear by itself.

Reply:
Yea, ... There are 2 commits at the beginning, that I wasn't aware of. ... strange.